src/main/include/controllers/DrivetrainController.hpp

```diff
@@ -300,6 +300,15 @@ class DrivetrainController : public ControllerBase<7, 2, 4> {
     static Eigen::Vector<double, 7> Dynamics(const Eigen::Vector<double, 7>& x,
                                              const Eigen::Vector<double, 2>& u);
 
+    /**
+     * The drivetrain system dynamics for turning in place.
+     *
+     * @param x The state vector.
+     * @param u The input vector.
+     */
+    static Eigen::Vector<double, 3> TurningDynamics(
+        const Eigen::Vector<double, 3>& x, const Eigen::Vector<double, 2>& u);
```

Comment:
The dynamics in this function are actually linear, so you could write a function that returns a `frc::LinearSystem<3, 2, 3>` instead.